### ISSUE TITLE
Update docs for CMake projects

### ISSUE THREAD

**Issue by SchoolyB** [SchoolyB/Cuazar#8]
After the fiasco of trying to implement Cuazar into [Hallpass Hero](https://github.com/SchoolyB/Hallpass-Hero) I ran into a issue. When building with CMake locally the `#include "path/to/lib"` could remain in source files no problem. But the moment I push the coded up to GH my workfows would give errors talking about variable and function re definitions in the source files. Can see what I mean [here](https://github.com/SchoolyB/Hallpass-Hero/pull/17#issuecomment-1833128005)

TLDR: Update docs  to recommend to folks commenting or removing the `#include "Cuazar/lib/Cuazar.h"` preprocessor  before pushing code.
